modules/category/category.graphql.js

```diff
@@ -4,6 +4,7 @@ const sharedFields = `
   name: [Language]
   images: ImageSet
   available: Boolean
+  translations_key: ID
```

Comment:
use camelCase instead of snake_case

Reply:
It's general approach in the project. And this field wrote in the same way in all the other components where it uses.

Reply:
Changed